src/log/list.js

```diff
@@ -2,35 +2,20 @@ const { default: API } = require('@aws-amplify/api-rest')
 
 const serializeAxiosResponse = require('../utils/serializeAxiosResponse')
 const { DEFAULT_REQ_OPTS } = require('../utils/defaultReqOpts')
-const { removeUndefinedKeys } = require('../utils/objUtils')
 
 /**
  * @desc queries the logs of a device
  * @memberof nexus.log
  * @function list
  * @async
  * @param {Object} params
```

Comment:
```suggestion
 * @param {Object} queryStringParameters, an object of query string parameters
```

my bad, this is the right syntax since we dont call the function with `nexus.log.list({queryStringParamters:{limit: 1})`
but with
`nexus.log.list({limit: 1}

Reply:
ahh ok